pkg/store/s3pg/reporting_store.go

```diff
@@ -33,25 +33,15 @@ func (s S3PGStore) GetReportingInfo(appID string) *upstreamtypes.ReportingInfo {
 	r := upstreamtypes.ReportingInfo{
 		InstanceID: appID,
 	}
-
-	clusterID, err := s.getClusterID()
-	if err != nil {
-		logger.Error(errors.Wrap(err, "failed to get cluster id"))
-	}
-	r.ClusterID = clusterID
 	configMap, err := getKotsadmIDConfigMap()
 	if err == nil && configMap != nil {
 		r.ClusterID = configMap.Data["id"]
 	} else if err == nil && configMap == nil {
 		//generate guid and use that as clusterId to identify that as a different install
-		clusterID = ksuid.New().String()
-		r.ClusterID = clusterID
-		_, err = CreateAdminIDConfigMap(clusterID)
-		if err != nil {
-			logger.Errorf("Failed to to create config map %v", err)
-		}
+		r.ClusterID = ksuid.New().String()
+		CreateAdminIDConfigMap(r.ClusterID)
 	} else {
-		logger.Errorf("Config map check error %v", err)
```

Comment:
we should still log this so that we know something went wrong

Reply:
Here we are replacing it with "generation of guid"  if incase k8 fails create configmap (as per our previous discussion).